### ISSUE TITLE
Add info about what to do with a project after joining the contrib group

### ISSUE THREAD

**Issue by alexfinnarn** [backdrop-ops/contrib#190]
I know this info was put into the issue where I applied, but I'm not seeing it here in the readme and don't know where one would find such information. Something like...

After You Join:
1. Clean up your project as much as possible.
2. Transfer the project to the backdrop-contrib organization following the steps mentioned in: [https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account/](https://help.github.com/articles/transferring-a-repository-owned-by-your-personal-account/)
3. Tag a release (link). You must tag a release in the format of "1.x-1.0.0". If you started your project with a title like "backdrop-port-of-xxx", feel free to drop the prefix or change the name to something you think makes more sense. 
4. Make sure you are subscribed to notifications for the project. If new issues are created, you need to know about them or else your project will be considered abandoned...something something...
5. (Optional) Subscribe to the issue queue on drupal.org if your module is a port. It's always nice to know about new features and bug fixes that you can incorporate in your module as well as important security-related issues. Similarly, if you fix a major issue or add a new feature, it's nice to let the Drupal maintainer know about this.

**Comment by jenlampton** [comment]
Yes, this is a great idea. thanks!  Would you like to make a PR against the readme and squish it in somewhere?

**Comment by alexfinnarn** [comment]
Okay. I added basically what I had written above. Feel free to change or add more Dad jokes :)

**Comment by jenlampton** [comment]
Thanks @alexfinnarn !  I've added two small comments to the PR. One other thought, since I want the 'Backdrop Contributed Project Agreement' to be high on the page, can we move this new section below it, or maybe even below 'Additional notes for non-coders' ?

**Comment by alexfinnarn** [comment]
Cool. I think I've made all the changes listed in the PR.

**Comment by klonos** [comment]
@alexfinnarn the new PR introduces numerous instances of multiple spaces and I think breaks the 80 character limit per row.

**Comment by alexfinnarn** [comment]
sorry, I was trying to edit via Github UI which obscures such things. PR updated.

**Comment by klonos** [comment]
@alexfinnarn what I tend to do is copy the text to a text editor that has a marker for the 80 characters limit (you can also consider one that has spell checking), I edit the text there, and when I'm ready, I copy/paste it back on GitHub 😉 

I approved the changes and also made a recommendation of how the paragraph about subscribing to the d.org issue queue should be better worded. Please review and let me know what you think about it. Thanx.

**Comment by BWPanda** [comment]
I'm in the process of transferring a module to backdrop-contrib, and Github's asking me about team access:

> Please select any teams you wish to have access to backdrop-contrib/featherlight.

I'll try to work out what to do here, but might be good to add this to the proposed steps...

**Comment by jenlampton** [comment]
That's a great point @BWPanda, please let us know if you need help figuring out what to do!

**Comment by BWPanda** [comment]
I just ticked both options ('authors' and 'security' IIRC). Still trying to figure out how to change from 'master' branch to '1.x-1.x' and then make a release...

**Comment by jenlampton** [comment]
The default branch is under "Settings".
<img width="1026" alt="screen shot 2017-09-07 at 7 58 14 pm" src="https://user-images.githubusercontent.com/397895/30194258-ed6ac0cc-9406-11e7-818d-bc53bf19c4e3.png">

**Comment by jenlampton** [comment]
Re-opening since we need more updates :)

**Comment by BWPanda** [comment]
@jenlampton I never saw an option to add myself as an admin... And I can't create a screenshot without transferring another repository over.

**Comment by jenlampton** [comment]
It's a blank "Collaborators" text field at the very bottom of the page, but when you type into it it autocompletes. Not very intuitive...  Maybe this would make a good video? I'll give that a shot :)

**Comment by BWPanda** [comment]
Here's what I see when I transfer a repository:

![screenshot-2017-9-11 bwpanda dummy](https://user-images.githubusercontent.com/2385329/30254853-5d783962-96e1-11e7-823b-e04ed254d64e.png)

No option to select myself as an admin...

Please now delete that repo: https://github.com/backdrop-contrib/dummy

**Comment by jenlampton** [comment]
Not in this section, as this is for "Teams".  The "Collaborators" section should be immediately below.

One thing to note is that you are a member of the "Authors" team so by transferring a project from yourself to Authors you are not necessarily removing your own access - as long as you grant Authors "Admin" privileges when you transfer.

**Comment by BWPanda** [comment]
Here is the entire process that I see. Hope this helps explain things...

![peek 2017-09-12 09-26](https://user-images.githubusercontent.com/2385329/30301218-c6c57036-979c-11e7-8b37-4e9e95778dcd.gif)

Can delete this again, thanks: https://github.com/backdrop-contrib/dummy2

**Comment by jenlampton** [comment]
Sigh, yes, this is a problem :( 

We have a bot that @Gormartsen has been working on that is supposed to "fix" all the permissions on transferred repositories, but AFAIK that hasn't been set up yet. 

Until that starts working feel free to open an issue in this queue for any project you need to have fixed. (And maybe we should add that to our new-maintainer notes as well, until we get the bot going.)

**Comment by BWPanda** [comment]
Ok, good to know what the issue is. I'll make a PR with updated instructions then.

**Comment by jenlampton** [comment]
> Ok, good to know what the issue is.

I'm not sure we've sorted out the issue yet, but we do at least know it's a problem and are planning on fixing it using the GitHub API, since apparently the UI can't handle it. :(

I did make one change to the group settings after watching this video, but I can't test it since I'm the organization owner... hm. Maybe I can test it as the GitHub bot? ...

**Comment by BWPanda** [comment]
If you need me to try transferring again to test anything, let me know.

**Comment by jenlampton** [comment]
ooh! that worked for dragonbot!  Can you try it again?

**Comment by BWPanda** [comment]
That worked! Same as last time (see above video), but this time the Settings option was available once I visited the transferred repo. I was listed as an admin, however Authors and Security only had Read access (had to manually change to Admin and Write respectively).

**Comment by jenlampton** [comment]
Well, one step in the right direction :) I'm not sure how to set the intial access level for the teams... I looked all over and couldn't find anything. Maybe we'll need the bot for that.

**Comment by BWPanda** [comment]
Looks like these still need deleting:
https://github.com/backdrop-contrib/dummy3
https://github.com/backdrop-contrib/egg

**Comment by jenlampton** [comment]
deleted, thanks! :)

**Comment by BWPanda** [comment]
Any thoughts on my PR above?

**Comment by jenlampton** [comment]
Merged!
